lobby/src/main/java/org/triplea/lobby/server/db/UserController.java

```diff
@@ -35,7 +35,8 @@ public HashedPassword getPassword(final String username) {
   private HashedPassword getPassword(final String username, final boolean legacy) {
     try (Connection con = connection.get();
         PreparedStatement ps = con
-            .prepareStatement("select password, coalesce(bcrypt_password, password) from ta_users where username=?")) {
+            .prepareStatement(
+                "select password, coalesce(bcrypt_password, password) from lobby_user where username=?")) {
```

Comment:
I made it a bit more consistent: d3d160b8b
It does not bother me too much, with existing code it's probably better in a different PR to avoid adding too many different kinds of changes to this PR. Regardless, it's a bit more consistent now